docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md

```diff
@@ -0,0 +1,51 @@
+# TAD-0002 TAD Reporting Standard
+
+## Context
+
+The TAD Reporting Standard proposes a standardised way of capturing information of ML-models and systems.
+
+## Assumptions
+
+There does not exists a standard of capturing all relevant information on ML-models that also includes
+fairness and bias tests and regulatory assessments.
+
+A widely used implementation for [Model Cards for Model Reporting](https://arxiv.org/abs/1810.03993) is given
+by the [Hugging Face Model Card metadata specification](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1),
+which in turn is based on [Papers with Code Model Index](https://github.com/paperswithcode/model-index).
+This implementation does not include fields for bias and fairness tests or regulartory assessments.
+
+## Decision
+
+We decided to implement a custom reporting standard. Our reporting standard can be split up
+into three elements.
+
+1. System Card, containing information about a group of ML-models which accomplish a specific task.
+A System Card can refer to multiple Model Cards, either a Model Card specified by the TAD Reporting
+Standard, or any other model card. A System Card can refer to multiple Assessment Cards.
+2. Model Card, containing information about a specific ML-model.
```

Comment:
I wonder if we also make measurements and metrics separate includes, so results in their own file, as we also do with assessments.

Reply:
We may want to, in general, accept includes, and say something like that here too.

This is what we currently say [in our standard](https://minbzk.github.io/ai-validation/Projects/TAD/Reporting-Standard/#introduction):

> These model_cards and assessment_cards can be included verbatim into a system_card, or referenced with an !include statement, allowing for minimal cards to be compact in a single file. Extensive cards can be split up for readability and maintainability. Our standard allows for the !include to be used anywhere.

Reply:
In principle you can !include inside the system card any field you want. The standard prefers to do this for in any case the model card and assessment card. If we also want to prefer to include metrics and measurements via an include mechanism, we should specify this in the TAD standard. So for this ADR I would not make these suggestions.

Reply:
Ok, awesome, should have seen that ;)